Client/Extensions/FileManagerExtension.swift

```diff
@@ -143,3 +143,23 @@ extension FileManager.SearchPathDirectory {
         return FileManager.default.urls(for: self, in: .userDomainMask).first
     }
 }
+
+extension FileManager {
+    
+    /// Navigates to download Folder inside the application's folder
+    func openBraveDownloadsFolder(_ completion: @escaping (Bool) -> Void) {
+        do {
+            let downloadsPath = try FileManager.default.downloadsPath()
+            
+            let folderPath = downloadsPath.absoluteString.replacingOccurrences(of: "file://", with: "shareddocuments://")
+            guard let braveFolderURL = URL(string: folderPath) else { return }
```

Comment:
If we're already having to re-create the `URL` perhaps we should replace the scheme in a `URLComponents` instead. Also you don't call `completion` on the url failing to create

I.e.

```swift
var components = URLComponents(
    url: try FileManager.default.downloadsPath(), 
    resolvingAgainstBaseURL: false
)
components.scheme = "shareddocuments"
guard let braveFolderURL = components.url else { 
  completion(false)
  return 
}
```

Reply:
Fix at https://github.com/brave/brave-ios/pull/4769/commits/e5a197beb089bc9fb747bbc94009e0edd161444e